components/database/mariadb-103/mariadb-103.p5m

```diff
@@ -0,0 +1,247 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2013 Alexander Pyhalov.  All rights reserved.
+# Copyright 2016-2017, Wiselabs Software Ltda - http://www.wiselabs.com.br . All rights reserved.
+# Copyright 2018, Michal Nowak
+#
+
+set name=pkg.fmri \
+    value=pkg:/database/mariadb-103@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary \
+    value="MariaDB 10.3 Database Management System"
+set name=info.classification \
+    value=org.opensolaris.category.2008:Development/Databases
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability committed>
+
+license mariadb-103.license license=GPLv2
+
+dir path=var/mariadb group=mysql mode=0700 owner=mysql
+dir path=var/mariadb/10.3 group=mysql mode=0700 owner=mysql
+dir path=var/mariadb/10.3/data group=mysql mode=0700 owner=mysql
+file files/my-medium.cnf path=etc/mariadb/10.3/my.cnf \
+    mode=0644 preserve=renamenew
+file files/auth_attr path=etc/security/auth_attr.d/mariadb-103
+file files/prof_attr path=etc/security/prof_attr.d/mariadb-103
+file files/mariadb_103.xml \
+    path=lib/svc/manifest/application/database/mariadb_103.xml
+file files/mariadb_103 path=lib/svc/method/mariadb_103 mode=0555
+
+link path=etc/mysql/my.cnf target=../mariadb/10.3/my.cnf mediator=mysql mediator-version=10.3 mediator-implementation=mariadb
```

Comment:
If we want to continue shipping MariaDB 10.1 by default (and we likely do), it's worth to add 
```
<transform link mediator=mysql  mediator-implementation=mariadb mediator-version=10.1 -> default mediator-priority vendor>
``` 
to transforms/defaults and bump MariaDB 10.1 COMPONENT_REVISION

Reply:
See https://github.com/OpenIndiana/oi-userland/pull/4361.